### DISCUSSION TITLE
SEB Client 1.5x Custom Start URL/Quit link not working

### DISCUSSION THREAD

**Discussion by imrhlrvndrn** [SafeExamBrowser/seb-win-refactoring#786] [Q&A]
We have recently begun experimenting with SEB Client for conducting restrictive exams that could be used as final exams for certification. As an EdTech company, we have our own custom learning management system where we administer all kinds of exams. It is now our intention to build an exam system that will be more serious in nature, and, therefore, a service such as SEB is our preferred choice.

## Current exam flow

1. The SEB Client is used for exams where the start URL in the "General" tab is a Google Sheet with questions. Students make a copy of that file and work on their solution.

1. As part of the same question sheet, we provide a Google Form link where the student can submit his/her details and a Google Sheet link at the end of the exam.

## A new approach

1. Recently, we decided to approach this with a different strategy where before starting the exam we would redirect students to our custom standalone portal where the students would submit basic details which they earlier submitted on Google Forms & start the exam.

1. After submitting we'll send an OTP like token to the student.

1. Exams are working on their solution in Google Sheets. When they quit, they'll be redirected to the same portal where they'll enter the token to authenticate & securely end the exam.

The above approach allows us to collect data about when students start/end their exams.

## Challenges

Unfortunately, we tried multiple alternatives to achieve the results

1. First we thought the URL might be an issue, so we tried both `https://www.*.com` & `https://*.com` ( for both Custom Start & Quit Link )

1. Then we thought maybe the quit password was the issue, so we removed the password

1. All that did was pop-up a confirmation dialog before quitting the exam

1. Then we tried removing the Start URL from the "General" tab, hoping it'd fallback to the custom URL. But it just took the default `https://safeexambrowser.org/start`

Are we missing something? Your guidance on this would mean a lot. Thanks in advance

**Comment by dbuechel** [comment]
The start URL needs to be a valid URL of an existing resource, see https://safeexambrowser.org/windows/win_usermanual_en.html#GeneralPane. For issues with the URL filter, you'd want to either first deactivate the URL filter and ensure everything works as expected or then activate the live application log and check which URLs are being blocked (or not), see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane > Allow Access to Application Log.
